### ISSUE TITLE
Error using PDB and CATH50 downloaded databases

### ISSUE THREAD

**Issue by Fede112** [steineggerlab/foldseek#294]
## Expected Behavior
Hi,
I am trying to use the CATH50 and PDB databases downloaded using the `foldseek databases` module to perform a straightforward search+convertalis with a self-made database containing around 30,000 queries.

## Current Behavior
The foldseek convertalis command fails with the following error:
```free(): invalid next size (normal)```

I managed to run the search by using cath50_seq instead of cath50. I did this because I noticed a problem in the indexing, but I am not sure if this is the correct approach. Is this okay?

## Your Environment
Foldseek Version: d326ca0212e48b2dfced1c1d9cd0d05b80b159b6
Also encountered the problem with version: foldseek/8-ef4e960

Thanks in advance

**Comment by milot-mirdita** [comment]
Could you please post the full terminal output of foldseek? 

Proceeding with `cath50_seq` is fine, this database contains all structures. the `cath50` contains only the representative structures, which can be expanded to the cluster members with `--cluster-search 1`.

But it shouldn't crash in either case.

**Comment by milot-mirdita** [comment]
Does it work correctly when you turn around the roles of query and target (i.e. make pdb the target db)

**Comment by Fede112** [comment]
yes, if I turn around the roles everything works just fine for either intel or arm.

**Comment by milot-mirdita** [comment]
We intended for all of these databases from `foldseek databases` to be used as target databases. I guess we have a bug when using them on the query side.

**Comment by martin-steinegger** [comment]
I just tried this and it did not result in an error:
```
foldseek databases PDB pdb tmp2
foldseek easy-search pdb ../example aln tmp -e 0.1 -a --max-seqs 30000  -s 4
```
